### PR TITLE
[ID] Add `Community/osu!monthly`

### PR DESCRIPTION
**Pull request by wowcake**
Translated osu!monthly wiki (https://osu.ppy.sh/wiki/en/Community/osu%21monthly) to Indonesia

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Community/osu!monthly/id.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+| Nama | Tanggal publikasi |
+| :-- | :-- |
+| [osu!monthly #1](https://osumonthly.com/pdf/osu!monthly-1.pdf) | 2011-03-05 |
+| [osu!monthly #2](https://osumonthly.com/pdf/osu!monthly-2.pdf) | 2011-05-06 |
+| [osu!monthly #3](https://osumonthly.com/pdf/osu!monthly-3.pdf) | 2011-06-07 |
+| [osu!monthly #4](https://osumonthly.com/pdf/osu!monthly-4.pdf) | 2011-07-07 |
+| [osu!monthly #5](https://osumonthly.com/pdf/osu!monthly-5.pdf) | 2011-08-15 |
+| [osu!monthly #6](https://osumonthly.com/pdf/osu!monthly-6.pdf) | 2011-10-27 |
+
+Informasi lebih banyak tentang versi osu!monthly yang pertama dapat ditemukan di [website JJ](https://web.archive.org/web/20201112030400/http://jjwanda.com/projects/osumonthly.html).
```
Comment (Nivalyx) [inline wiki/Community/osu!monthly/id.md:30]:
```suggestion
Informasi lebih lanjut mengenai musim osu!monthly pertama dapat dilihat pada [situs web milik JJ berikut](https://web.archive.org/web/20201112030400/http://jjwanda.com/projects/osumonthly.html).
```

File: wiki/Community/osu!monthly/id.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+Terbitan osu!monthly yang pertama dan paling awalnya dibuat oleh [jjrocks](https://osu.ppy.sh/users/30985), [Corin](https://osu.ppy.sh/users/639270), dan temannya dari Maret 2011 sampai Oktober 2011. Artikelnya juga dirilis dalam format PDF dengan informasi yang penting tentang komunitas osu!.
+
+Terinspirasi oleh versi osu!monthly yang pertama, tim baru telah dibentuk berkat karena [mangomizer](https://osu.ppy.sh/users/1893718) dan [clayton](https://osu.ppy.sh/users/3666350), dan iterasi osu!monthly yang baru pun dapat dipublikasikan lagi dari Oktober 2017 sampai Februari 2018. Formatnya juga masih PDF, sama seperti dulu, tetapi dengan desain indah yang lebih konsisten dan menyegarkan.
+
+Dengan keinginan untuk menghidupkan kembali proyek ini, para [tim osu!monthly](/wiki/People/osu!monthly_team) akhirnya dibentuk oleh [0x84f](https://osu.ppy.sh/users/7944724), menghidupkan kembali proyek tersebut dan sekaligus juga memimpinnya, kali ini, langsung sebagai artikel berita di laman osu!. Proyek ini saat ini sedang berlangsung.
+
+## Arsip
+
+### Season 1 (2011)
```
Comment (Nivalyx) [inline wiki/Community/osu!monthly/id.md:19]:
```suggestion
### Musim pertama (2011)
```

File: wiki/Community/osu!monthly/id.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+| [osu!monthly #1](https://osumonthly.com/pdf/osu!monthly-1.pdf) | 2011-03-05 |
+| [osu!monthly #2](https://osumonthly.com/pdf/osu!monthly-2.pdf) | 2011-05-06 |
+| [osu!monthly #3](https://osumonthly.com/pdf/osu!monthly-3.pdf) | 2011-06-07 |
+| [osu!monthly #4](https://osumonthly.com/pdf/osu!monthly-4.pdf) | 2011-07-07 |
+| [osu!monthly #5](https://osumonthly.com/pdf/osu!monthly-5.pdf) | 2011-08-15 |
+| [osu!monthly #6](https://osumonthly.com/pdf/osu!monthly-6.pdf) | 2011-10-27 |
+
+Informasi lebih banyak tentang versi osu!monthly yang pertama dapat ditemukan di [website JJ](https://web.archive.org/web/20201112030400/http://jjwanda.com/projects/osumonthly.html).
+
+### Season 2 (2017–2018)
```
Comment (Nivalyx) [inline wiki/Community/osu!monthly/id.md:32]:
```suggestion
### Musim kedua (2017–2018)
```

File: wiki/Community/osu!monthly/id.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+| Nama | Tanggal publikasi |
+| :-- | :-- |
+| [osu!monthly #1](https://osumonthly.com/pdf/osu!monthly-2017-10.pdf) | 2017-11-04 |
+| [osu!monthly #2](https://osumonthly.com/pdf/osu!monthly-2017-11.pdf) | 2017-12-26 |
+| [osu!monthly #3](https://osumonthly.com/pdf/osu!monthly-2018-02.pdf) | 2018-02-17 |
+
+Informasi lebih banyak tentang versi osu!monthly yang kedua dapat ditemukan di [website resmi](https://osumonthly.com/), yang mana juga mengandung daftar tim dan arsip-arsip artikel.
+
+### Season 3 (2022–kini)
```
Comment (Nivalyx) [inline wiki/Community/osu!monthly/id.md:42]:
```suggestion
### Musim ketiga (2022–sekarang)
```

**Review by Nivalyx — COMMENTED** [review]
will check the rest a bit later i guess

**Comment by Nivalyx** [comment]
okay @wowcake i think that's all

**Comment by wowcake** [comment]
applied, once again, thanks as always @Nivalyx 🌻

**Review by TicClick — APPROVED** [review]

**Pull request merged**